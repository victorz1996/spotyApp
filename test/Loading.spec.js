import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Bootstrapvue from 'bootstrap-vue'
import loading from '../components/Loading.vue'

let wrapper
beforeEach(() => {
  Vue.use(Bootstrapvue)
  wrapper = shallowMount(loading)
})

describe('Loading', () => {
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it('HTMl renderizado structure esperada', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
