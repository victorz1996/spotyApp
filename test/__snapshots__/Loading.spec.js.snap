// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading HTMl renderizado structure esperada 1`] = `
<div
  class="d-flex justify-content-center mb-3"
>
  <b-spinner-stub
    label="Large Spinner"
    role="status"
    style="width: 3rem; height: 3rem;"
    tag="span"
    type="border"
  />
</div>
`;
